src/mapper/pkg/kubefinder/kubefinder.go

```diff
@@ -442,3 +463,53 @@ func (k *KubeFinder) ResolveOtterizeIdentityForService(ctx context.Context, svc
 	dstSvcIdentity.KubernetesService = lo.ToPtr(svc.Name)
 	return dstSvcIdentity, true, nil
 }
+
+func (k *KubeFinder) IsSrcIpLocal(ctx context.Context, ip string) (bool, error) {
+	isNode, err := k.IsNodeIP(ctx, ip)
+	if err != nil {
+		return false, errors.Wrap(err)
+	}
+	if isNode {
+		return true, nil
+	}
+
+	isPod, err := k.IsPodIp(ctx, ip)
+	if err != nil {
+		return false, errors.Wrap(err)
+	}
+	if isPod {
+		return true, nil
+	}
+
+	_, isControlPlane, err := k.ResolveIPToControlPlane(ctx, ip)
+	if err != nil {
+		return false, errors.Wrap(err)
+	}
+	if isControlPlane {
+		return true, nil
+	}
+
+	//TODO: Should we check for services/endpoints as well?
```

Comment:
Options for expanding this function (and making it less likely that we will miss detecting something as incoming traffic)

Reply:
rather than phrasing this as TODOs (which sounds like we were just lazy), document it as "known issue" / "points for improvement". I'd write it like that:
```
Known issue: this function is currently missing support for services/endpoints, Pod CIDR, ...). 
```